src/lib/components/NewLayeredMap/components/Controls/components/MousePosition.js

```diff
@@ -87,7 +87,7 @@ const MousePosition = (props) => {
         const onScreenCanvas = focusedImageLayer.getCanvas ? focusedImageLayer.getCanvas() : null;
         const minvalue = (focusedImageLayer.options || {}).minvalue;
         const maxvalue = (focusedImageLayer.options || {}).maxvalue;
-        if(!url || !onScreenCanvas || !minvalue || !maxvalue) {
+        if(!url || !onScreenCanvas || minvalue == null || !maxvalue) { // TODO fix minvalue = 0
```

Comment:
Should be `===` and also need to check for undefined. Maybe a type-check if enough:

```
typeof minvalue === "number"
```